hamocc/aufr_bgc.F90

```diff
@@ -163,18 +163,19 @@ SUBROUTINE AUFR_BGC(kpie,kpje,kpke,ntr,ntrbgc,itrbgc,trc,               &
       !
       testio=0
       IF(mnproc==1 .AND. IOTYPE==0) THEN
+      
         ncstat = NF90_OPEN(rstfnm,NF90_NOWRITE, ncid)
         IF ( ncstat .NE. NF90_NOERR ) THEN
-             CALL xchalt('(AUFR: Problem with netCDF1)')
-                    stop '(AUFR: Problem with netCDF1)'
+           CALL xchalt('(AUFR: Problem with netCDF1)')
```

Comment:
Admittedly, the indentation of the stop is unclear to me (if it is about aligning the message, you could just shift the message). Go as you like.

Reply:
Agree, will fix this.